collections/Laud_Misc/MS_Laud_Misc_133.xml

```diff
@@ -57,54 +81,97 @@
-                        <author key="person_199159880"><!-- check: and/or 231793925 -->Evodius Uzalitanus</author>
-                        <title key="work_1520">Letter</title>
-                        <textLang mainLang="la">Latin</textLang>
+                     <msItem>
+                        <locus>(fols. 83v–85r)</locus>
+                        <author key="person_199159880">Evodius Uzalitanus</author>
+                        <title key="work_1520">Epistula ad Valentinum abb. Hadrumetinum</title>
                      </msItem>
-                     <msItem n="6" xml:id="MS_Laud_Misc_133-part1-item6">
+                     <msItem>
+                        <locus>(fols. 85r–v)</locus>
                         <author key="person_170225">Celestine I</author>
-                        <title key="work_1220">Letter</title>
-                        <note>(ep. 21)</note>
-                        <textLang mainLang="la">Latin</textLang>
+                        <title key="work_1220">Epistula ad episcopos Galliarum</title>
+                        <bibl>ep. 21</bibl>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 85v–88r)</locus>
+                        <author key="person_52485706">Prosper of Aquitaine</author>
+                        <title key="work_17509">Praeteritorum sedis apostolicae episcoporum auctoritates de gratia Dei</title>
                      </msItem>
-                     <msItem n="7" xml:id="MS_Laud_Misc_133-part1-item7">
+                     <msItem>
+                        <locus>(fols. 88r–89r)</locus>
                         <author key="person_7386286">Ps.-Augustine</author>
-                        <title key="work_3873">Commonitorium quomodo sit agendum cum manichaeis</title>
-                        <textLang mainLang="la">Latin</textLang>
+                        <title key="work_3873">Commonitorium quomodo sit agendum cum Manichaeis qui confitentur</title>
                      </msItem>
                   </msContents>
                   <physDesc>
                      <objectDesc form="codex">
                         <supportDesc material="perg">
                            <support>parchment</support>
                         </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="29">Ruled in drypoint, two columns of 29 lines, ruled space <dimensions type="ruled" unit="mm">
+                                 <height>220</height>
+                                 <width>170</width>
+                              </dimensions>
+                           </layout>
+                        </layoutDesc>
                      </objectDesc>
                      <decoDesc>
                         <decoNote>Border (marginal sketch, fol. 143v), added in the 12th century, first half. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 1301)</decoNote>
```

Comment:
This decoNote belongs with part 2. Not sure if the plain initials in part 1 deserve mention in their own right?